sys/include/crypto/ciphers.h

```diff
@@ -104,6 +104,7 @@ typedef struct cipher_interface_st {
                    uint8_t *plain_block);
 } cipher_interface_t;
 
+/** Pointer type to BlockCipher-Interface for the Cipher-Algorithms */
```

Comment:
I think that you either need to add `@brief` or use the postfix notation with `/**<` to be consistent

Reply:
This is the one liner equivalent to `@brief`. `/**< */` can be used at the end of the same line.

Reply:
Autobrief is set to yes so this comment is perfectly valid IMHO:

https://github.com/RIOT-OS/RIOT/blob/fadb4d9e6ce35d707555127504791185adb6af47/doc/doxygen/riot.doxyfile#L173-L180